src/locales/en/town.js

```diff
@@ -617,9 +617,9 @@ export const townChapter5 = {
     "...", // matilda
     "What's wrong?", // ann
     "Uh, yeah yeah...", // matilda
-    "I'm finally at peace.", // matilda
-    "Really!?", // ann
-    "I'm very happy about something.", // matilda
+    "We're finally at peace.", // matilda
+    "Yeah, we are!", // ann
+    "And I'm very happy about another something now.", // matilda
```

Comment:
She has two reasons for why she is happy.
- The king beat the dragon
- She talked with Annabelle

Matilda talking about the "happy" by the first reason in these 3 lines,
because every citizen should talk about that at first topic.

Reply:
622: "And I'm very happy about something else now too.",